### DISCUSSION TITLE
Paperless NGX

### DISCUSSION THREAD

**Discussion by Maggo1977** [tteck/Proxmox#3041] [Help Needed]
Hallo,
ich habe auf meinem Proxmox PVE versucht mit Hilfe des fertigen Helfer-Scripts Paperless-NGX zu installieren. Das ganze hat soweit auch funktioniert. Ich kann über die IP und den Port: 8000 auch die Weboberfläche aufrufen.
Hier wird dann ein Benutzername und ein Passwort verlangt. Jedoch habe ich dieses nicht. Die Eingabe von admin:admin bleibt erfolglos. Habe ich etwas falsch gemacht? Was kann ich tun?

**Comment by tteck** [comment]
English only, please.

**Reply by Maggo1977** [reply]
I tried to install Paperless-NGX on my Proxmox PVE using the ready-made helper script. The whole thing worked so far. I can also access the web interface via the IP and port: 8000.
A username and password will then be required here. However, I don't have this. Entering admin:admin remains successful. Did I do something wrong? What can I do?

**Reply by tteck** [reply]
![Screenshot 2024-05-23 7 33 04 AM](https://github.com/tteck/Proxmox/assets/21342670/391ac436-b63a-494b-bc5d-64a0520ac713)

**Reply by Maggo1977** [reply]
I did that as per the instructions. Unfortunately there are no login details there.

**Reply by tteck** [reply]
❗ It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.

**Comment by Maggo1977** [comment]
Thanks..... I have it
